project/Win32BuildSetup/getdeploydependencies.bat

```diff
@@ -9,6 +9,11 @@ PUSHD %~dp0
 
 if not exist %DOWNLOAD_FOLDER% mkdir %DOWNLOAD_FOLDER%
 
+:: Remove file if is older of 30 days - refresh Visual C++ Redistributable
+if exist %DOWNLOAD_FOLDER%\%DOWNLOAD_FILE% (
+  forfiles /P %DOWNLOAD_FOLDER% /D -30 /C "cmd /c del @path" 2>nul
```

Comment:
Out of curiosity, the following sites mentions a bug with the /c command

https://ss64.com/nt/forfiles.html

```
Recent versions of FORFILES contain a bug, in processing command line arguments: running a /c command that contains an argument such as FORFILES /C "PING -a" will fail. The expected convention is that argv[0] will contain the program name, but FORFILES instead passes the first argument as argv[0]. Old versions e.g. FORFILES version 1.46 (24th March 2006) do not suffer from this bug.

One workaround for this is to include the program name as the first argument, this will be passed along by FORFILES and the CreateProcess call will then work as expected:
FORFILES /c "Ping Ping -a"
```

That relevant these days?

Reply:
Tested locally in Windows 11 23H2 and is working correctly (for command passed at least).

Anyway same page says that this bug not exists when is used "cmd /c xxx" to call command instead of call directly:

> Alternatively pass the command to [CMD /C](https://ss64.com/nt/cmd.html), which will then process all the arguments correctly.
> If you want to use an internal commmand, or if you want to use redirection, pipe, command concatenation, etc, then you must use CMD /C.
> FORFILES /c "CMD /C Ping -a"

Which is the way it is used here because it is extensively documented in many examples.